README.md

```diff
@@ -11,46 +10,69 @@
+
 - C-callable library interface
 - Rust crate
 
 Ursa is divided into two sub libraries: libursa and libzmix.
 
 ## Libursa
 
-Designed for cryptographic primitives like simple digital signatures, encryption schemes, and key exchange.
+Designed for cryptographic primitives like simple digital signatures, encryption
+schemes, and key exchange.
 
 ## Libzmix
-A generic way to create zero-knowledge proofs, proving statements about multiple cryptographic building blocks, containing signatures, commitments, and verifiable encryption. Libzmix uses many of the building blocks found in Libursa.
+
+A generic way to create zero-knowledge proofs, proving statements
+about multiple cryptographic building blocks, containing signatures,
+commitments, and verifiable encryption. Libzmix uses many of the building blocks
+found in Libursa.
 
 # Dependencies
 
 Ursa and zmix use the following external dependencies:
 
-- [libsodium 1.0.14](https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz) (Written in C)
-- [openssl 1.1.0j](https://www.openssl.org/source/openssl-1.1.0j.tar.gz) or newer (Written in C)
+- [libsodium
+  1.0.14](https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz)
+  (Written in C)
+- [openssl 1.1.0j](https://www.openssl.org/source/openssl-1.1.0j.tar.gz) or
+  newer (Written in C)
 - [libsecp256k1](https://github.com/bitcoin-core/secp256k1) (Written in C)
 
-These dependencies are used when building in the default secure mode. These libraries are widely known.
-There is a goal to be able to compile Ursa from rust only code for portability reasons like generating web assemblies without
-the worry of compatibility issues from C code. For this reason, Ursa can be compiled with *portable* mode which replaces any external
-libraries with rust compatible code. Ursa developers take care when choosing suitable replacements that are cryptographically safe to use
-but may not have been audited and vetted in a similar manner to these external libraries. Ursa consumers should note this when using
-portable mode for their applications.
+These dependencies are used when building in the default secure mode. These
+libraries are widely known.  There is a goal to be able to compile Ursa from
+rust only code for portability reasons like generating web assemblies without
+the worry of compatibility issues from C code. For this reason, Ursa can be
+compiled with *portable* mode which replaces any external libraries with rust
+compatible code. Ursa developers take care when choosing suitable replacements
+that are cryptographically safe to use but may not have been audited and vetted
+in a similar manner to these external libraries. Ursa consumers should note this
+when using portable mode for their applications.
 
 # Building from Source
-Libursa and Libzmix rely on libsodium for the default secure mode. Please see the following document for specific
-platform installations [here](docs/build-enviroment.md).
```

Comment:
Lets spell our filenames correctly. Move `docs/build-enviroment.md` to `docs/build-environment.md` and link to the correctly-spelled name.

Reply:
Fixed